.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## What this PR does?
+
+<!-- Please describe what your changes does. -->
+
+## How to test your changes?
+
+<!-- Describe how we can test your changes? Is there any notable behaviour
+     that the end users could notice? -->
+
+## Related issue number
+
+<!-- Is there any issue opened that get fixed by this? -->
+
+## Checklist
+
+- [ ] Code is written
+- [ ] Tests are added
+- [ ] Documentation reflects your changes
+- [ ] CHANGES.txt is updated
```

Comment:
I don't want to force the bullet.
Updating CHANGES.txt is not trivial: the file most likely conflicts with other commits.
For now I update CHANGES after every merge manually. Let's keep it up by committers.

In CPython resolving conflicts in CHANGES is the most pain in the ass.
IIRC @brettcannon proposed to collect CHANGES from CPython issue tracker -- it's cool but we don't need it for such small projects as aiohttp with 4 guys with commit rigths for now.